crates/trigger/Cargo.toml

```diff
@@ -32,7 +32,9 @@ spin-factor-wasi = { path = "../factor-wasi" }
 spin-factors = { path = "../factors" }
 spin-factors-executor = { path = "../factors-executor" }
 spin-telemetry = { path = "../telemetry" }
+spin-variables = { path = "../variables" }
 tokio = { workspace = true, features = ["fs", "rt"] }
+toml = { workspace = true }
```

Comment:
I think Ryan had tried to push TOML dependencies into the runtime config crate, so that the trigger crate was abstracted from the physical representation of the runtime config?  (For e.g. SpinKube scenarios.)  I am not sure though.